code/__DEFINES/achievements.dm

```diff
@@ -77,7 +77,6 @@
 #define BOSS_MEDAL_HIEROPHANT "Hierophant Killer"
 #define BOSS_MEDAL_LEGION   "Legion Killer"
 #define BOSS_MEDAL_TENDRIL   "Tendril Exterminator"
-#define BOSS_MEDAL_SWARMERS   "Swarmer Beacon Killer"
```

Comment:
I think these defines should stay, just like king goat ones.

Reply:
done